### PR TITLE
Add progress_indicators 0.0.1... and 0.0.2 

### PR DESCRIPTION
**Pull request by pyjarrett**

### CODE REVIEW

File: index/pr/progress_indicators/progress_indicators-0.0.1.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name = "progress_indicators"
+description = "Ways of displaying progress in command line tools."
+version = "0.0.1"
+website = "https://github.com/pyjarrett/progress_indicators"
+authors = ["Paul Jarrett"]
+licenses = "Apache-2.0"
+
+maintainers = ["Paul Jarrett <jarrett.paul.young@gmail.com>"]
+maintainers-logins = ["pyjarrett"]
+tags = ["progress"]
```
Comment (mosteo) [inline index/pr/progress_indicators/progress_indicators-0.0.1.toml]: What do you think of having "command-line" as an extra tag?

**Review by mosteo — COMMENTED** [review]
Looks good to me, see my comment about an extra tag.

File: index/pr/progress_indicators/progress_indicators-0.0.1.toml (hunk 1 above)
Comment (pyjarrett) [reply index/pr/progress_indicators/progress_indicators-0.0.1.toml]: I actually had that in there originally as "command line" but it complained so I removed it.  I've added it back with the hyphen.

**Comment by pyjarrett** [comment]
I've had to fix some usability things due to some preliminary testing from @onox (thanks onox!), so I'm going to cut this and resubmit.

**Pull request closed without merging**